pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -178,7 +178,27 @@ type FailoverBehavior struct {
 	// Cluster indicates failover behaviors in case of cluster failure.
 	// If this value is nil, failover is disabled.
 	// +optional
-	// Cluster *ClusterFailoverBehavior `json:"cluster,omitempty"`
+	Cluster *ClusterFailoverBehavior `json:"cluster,omitempty"`
```

Comment:
What is the difference between this and the Failover that we currently have?

Reply:
Currently we have the failover capability, and we may need to open this cap to users for fine-grained configuration in the next step.